docs/components/Address.md

```diff
@@ -31,20 +31,20 @@ There are also custom output properties which can be used to reference component
 - **Theme - Theme** - Configures the styling of component elements
     - **Default:** - See [Theme Documentation](../Theme.md).
 
-The following input properties are records that control whether the respective text input is required or optional, controls the visibility of the respective field and controls the width proportion of the text input. Note, a width proportion of 1 results in the text input width equal to the width of the component.
+The following input properties are records that control whether the respective text input is required or optional, the visibility of the respective field, the width proportion of the text input, the default value of the respective input that a user sees when they navigate to the component and whether the respective text input is in edit or disabled mode. Note, a width proportion of 1 results in the text input width equal to the width of the component.
```

Comment:
This is getting a bit lengthy to read now, might read easier if it was split out into something like

```
The following input properties are records that configure the state of each input control, containing:
  - Required: Whether the input is required or optional
  - Visible: Whether the input is visible or hidden
...
```